app/controllers/tutors_controller.rb

```diff
@@ -0,0 +1,3 @@
+class TutorsController < ApplicationController
+  expose :profile
```

Comment:
Would be nice to add `scope` to restrict it to the users who have `tutor` role, something like:
`Profile.includes(:user).where(users: {role: :tutor})`